src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -12,7 +12,7 @@ class PrivacyModule(
 ) : Module {
     private val patterns: List<Regex> = listOf(
         """\(Session ID is token:""".toRegex(),
-        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex(),
+        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-^]{2,15}\b""".toRegex(),
```

Comment:
@chandler05 is it intended that `^` is allowed now in the top level domain regex? [RFC 952](https://tools.ietf.org/html/rfc952) does not mention that.

Reply:
Yes, it removes underscores from the regex as they aren't allowed in domains

Reply:
But why does it instead now add `^` as allowed character?

Reply:
Has been clarified, the regex range was added by accident (see also #539) and chandler wanted to preserve it.